pypolyclip/pypolyclip.py

```diff
@@ -126,87 +134,90 @@ def single(x, y, nxy):
     ----------
     x : int or float
         The x coordinates of the polygon corners
-    
+
     y : int or float
         The y coordinates of the polygon corners
 
     nxy : list, tuple, or `np.ndarray`
         The size of the pixel grid.
```

Comment:
```suggestion
           The size of the pixel grid.
```
The description lines should line up with line above.  This is an issue in several places.